front-end/src/components/Absence/Absence.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <div>
+    <input
+      ref="attachmentUpload"
+      type="file"
+      hidden
```

Comment:
?

Reply:
This input is used for attachment upload. Since we need only its functionality, it is hidden

Reply:
You sure that `ElementUI`'s [`upload`](https://element.eleme.io/#/en-US/component/upload#upload) shouldn't be used here?

Reply:
Current UI looks as follows. If we want to leave it as it is, than element's `upload `may be not the best option. It has some additional functionality, for instance file list below `upload `, that not needed at all. 

Solution with hidden input looks too crutchy? 
![image](https://user-images.githubusercontent.com/55359532/144219770-9148e376-e277-4b57-ad06-180531418ab8.png)

Reply:
Okay, let's leave this as is.